### PR TITLE
Implement Get anime list and Get single anime

### PR DESCRIPTION
**Pull request by Shiney-Kelly**
## Work Summary
- Build get entire anime list function
- Add headers to connect to API
- Add refresh functionality
- Add Picasso image auto resizing
- Add shared view models
- Build empty, error, loading states
- Enhanced UI by setting truncating texts
- Change brand color

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -15,6 +15,7 @@ android {
         versionCode 1
         versionName "1.0"
         buildConfigField("String", "API_BASE_PATH", apiBasePath)
```
Comment (Tony-Feng) [inline app/build.gradle:17]: this is going to be added in the github/local config as well

**Review by Tony-Feng — COMMENTED** [review]
and where's the bearer token prefix?

File: app/build.gradle (hunk 1 above)
Comment (Shiney-Kelly) [reply app/build.gradle:17]: config used in gradle.properties

**Comment by Shiney-Kelly** [comment]
> and where's the bearer token prefix?

In gradle.properties

File: app/build.gradle (hunk 1 above)
Comment (Tony-Feng) [reply app/build.gradle:17]: yes, and this will be part of config used github action

**Comment by Tony-Feng** [comment]
then LGTM!

**Review by Tony-Feng — APPROVED** [review]

**Pull request merged**